docs/components/arm/fake.md

```diff
@@ -34,20 +35,20 @@ Note that this visual example sets the `fake` arm to act as a `ur5e` arm.
     "model": "fake",
     "attributes": {
         "arm-model": "<your_arm_model>",
-        "model-path": "<your_arm_model_config_filepath>" // REMOVE if using arm-model
+        // "model-path": "<your_arm_model_config_filepath>"
```

Comment:
Why the "//"? If these two attributes are mutually exclusive, should we just show them both normally here and then explain that below?

Reply:
Hmmm good point-- I thought that would be a better way to provide a template for either. Arm-model is probably preferred for most users. It is explained down below pretty throughly IMO

Reply:
ok, removed!